src/docfx/serve/lsp/LanguageServerProgressReporter.cs

```diff
@@ -28,5 +28,10 @@ public void Report(string message)
                 Message = message,
             });
         }
+
+        public void OnCompleted()
+        {
+            _workDoneManager.OnCompleted();
```

Comment:
move this line to the `Dispose` method to let `using` automatically complete the `_workDoneManager`?